src/AGCertificateVerifier.h

```diff
@@ -189,11 +264,11 @@ class AGCertificateVerifier {
 
     AGVerifyResult verifyPins(const std::string &dnsName, STACK_OF(X509) *certChain);
```

Comment:
What are these pins? Are they updated? If not, please explain how pins are supposed to be updated